examples/Cassie/input_supervisor.cc

```diff
@@ -77,13 +94,23 @@ void InputSupervisor::SetMotorTorques(const Context<double>& context,
       is_error || (command->get_timestamp() -
                        context.get_discrete_state(prev_efforts_time_index_)[0] >
                    kMaxControllerDelay);
+  is_error =
+      is_error || context.get_discrete_state(soft_estop_flag_index_)[0] != 0;
```

Comment:
By adding the soft e-stop as an error we are setting the output command to 0 instead of the damping command aren't we?

Reply:
this is not obvious from the code and probably could be more obvious but since the soft e-stop check and override also has a return statement, the lines that zero out the controller commands don't get called.

Reply:
Ah I see it now. I couldn't tell because of how the intervening code was hidden.